SYCL/ESIMD/api/functional/ctors/ctor_array_core.cpp

```diff
@@ -32,7 +32,11 @@ int main(int, char **) {
 
   bool passed = true;
 
+#ifdef FULL_TYPE_COVERAGE
```

Comment:
```suggestion
#ifdef ESIMD_TESTS_FULL_TYPE_COVERAGE
```
Let's have some prefix specific for ESIMD API tests for every macro we define. What do you think?

```suggestion
#ifdef ESIMD_TESTS_FULL_COVERAGE
```
Also, what about not to limit the restrictions to the types? Less simd vector sizes covered, smaller sets of the input values - all these cases are handy to improve CI performance on the regular basis.

Reply:
>Also, what about not to limit the restrictions to the types? Less simd vector sizes covered

I thinks that decreasing types it's enough, but you suggest to increase the number of types that will be used

So I think that we can decrease used vector sizes. we can use 1 and 16 only

Reply:
I've applied this comment [here](https://github.com/intel/llvm-test-suite/pull/832/commits/e162bda82d57be4839dc67b58c109b6f7cf9b438#diff-3bbc1ab1445c827282cc08d73f76b58cf553d9ced2c799ba46e6148583896d22R297-R303)